### DISCUSSION TITLE
在流式上传至非本地的Driver时 也会把文件临时上传至服务器，但上传的文件在哪个文件夹下，如何修改它的位置？

### DISCUSSION THREAD

**Discussion by leeskyler-top** [AlistGo/alist#6823] [Q&A]
![1721452901217](https://github.com/user-attachments/assets/1ef1152e-5cec-4250-a21c-ca4d634ddb70)
![image](https://github.com/user-attachments/assets/04ae6879-5368-4cd9-a5ad-d7903bdf07ae)
![cfefbab208a18b8bb68b804a073f185](https://github.com/user-attachments/assets/2b05b3af-bcad-4965-994a-a3df2d5eaf77)
因为服务器硬盘剩余容量小，上传超容量的大文件或同时上传多个文件的总和大于了服务器硬盘剩余容量，就会导致任务失败，是否能考虑修改临时存储文件夹？
流式上传会不会产生文件？

**Comment by anwen-anyi** [comment]
https://alist.nn.ci/zh/config/configuration.html#temp-dir
